src/util.ts

```diff
@@ -0,0 +1,33 @@
+export const fetchVstorageKeys = (
```

Comment:
please note this issue to provide key queries in casting: https://github.com/Agoric/agoric-sdk/issues/6690

Reply:
Left a comment